### DISCUSSION TITLE
What's the correct way of adding a label to a SegmentedControl

### DISCUSSION THREAD

**Discussion by mattfullerdev** [mantinedev/mantine#4720] [Q&A] [answered]
I'm writing a form and wanted to use a SegmentedControl, what's the best way of adding a label to it? I wanted it in such a way so it's above the control and the same as a standard input label. I can add the text manually but I feel like there's probably an easier/better way of doing it, can I use the input wrapper somehow?

Thanks,

**Comment by rtivital** [comment, marked as answer]
See https://mantine.dev/core/input/#inputlabel-inputerror-and-inputdescription

**Reply by mattfullerdev** [reply]
Thanks!
